stripe-core/src/main/java/com/stripe/android/core/networking/RequestHeadersFactory.kt

```diff
@@ -63,22 +64,22 @@ sealed class RequestHeadersFactory {
                 return mapOf(
                     HEADER_ACCEPT to "application/json",
                     HEADER_STRIPE_VERSION to apiVersion,
-                    HEADER_AUTHORIZATION to "Bearer ${options.apiKey}"
+                    HEADER_AUTHORIZATION to "Bearer ${optionsProvider().apiKey}"
```

Comment:
could we make a single call to `val options = optionsProvider()` and then reference that throughout this method? In case the optionsProvider call somehow gets to be expensive, it would be good to reduce the number of calls to it.

Reply:
nice catch! updated